app/pages/admin/project-status/experimental-features.jsx

```diff
@@ -26,7 +26,8 @@ const experimentalFeatures = [
   'highlighter',
   'translator-role',
   'museum-role',
-  'transcription-task'
+  'transcription-task',
+  'wildcam classroom'  // Indicates a Project is linked to a "WildCam Lab"-type Zooniverse Classroom. Allows the classifier to select a workflow (i.e. "classroom assignment") directly via ID.
```

Comment:
Not a blocking issue: I wonder why we used dashes in some experimental tools but spaces in others?

Reply:
Don't forget camelCase in 'freehandSegmentLine' and Capitalised First Letters of 'Gravity Spy Gold Standard'.

I genuinely don't what the real reason is, but if I were to hazard a guess, it's because nobody bothered to standardise anything because these things, since they were ""experimental""? Or perhaps it's because each dev brought in their own coding assumptions to the repo. I know I recently made a mistake on monorepo (!) when I suggested that Erik create a "temporal-point" drawing tool, before I was corrected into using camelCase (i.e. "temporalPoint")